package.json

```diff
@@ -95,6 +95,7 @@
     "react-redux": "8.1.2",
     "react-simple-code-editor": "0.13.1",
     "react-ssr-prepass": "npm:preact-ssr-prepass",
+    "recharts": "^2.11.0",
```

Comment:
We made a decision long time ago to always pin dependency version. Please lmk if you have thoughts on that.

Reply:
Nope I agree